### PR TITLE
Retry setting up the postgres container

### PR DESCRIPTION
**Pull request by kalikiana**
Fixes: https://progress.opensuse.org/issues/110803

### CODE REVIEW

**Comment by kalikiana** [comment]
> Did you do a verification run?

Yes. But apparently `install_deps` is never run: https://openqa.opensuse.org/tests/2339622#step/setup_env/7

**Comment by kalikiana** [comment]
> it probably needs to be installed the same way as docker in tests/install/openqa_webui.pm
> 
> ```
> assert_script_run("zypper --non-interactive install docker git", timeout => 600);
> ```

That makes a lot of sense in hindsight since it's run as part of the test 🤦🏾 Thanks

File: tests/install/openqa_webui.pm [hunk 1]
```diff
@@ -53,7 +53,7 @@ EOF
 }
 
 sub install_containers {
-    assert_script_run("zypper --non-interactive install docker git", timeout => 600);
+    assert_script_run("zypper --non-interactive install docker git podman", timeout => 600);
```
Comment (Martchus) [inline tests/install/openqa_webui.pm:56]: Have you checked whether we still use docker elsewhere? If no, I suppose docker can be removed from here.

**Comment by kalikiana** [comment]
> That makes a lot of sense in hindsight since it's run as part of the test 🤦🏾 Thanks

Successful verification run: https://openqa.opensuse.org/tests/2339705

File: tests/install/openqa_webui.pm (hunk 1 above)
Comment (kalikiana) [reply tests/install/openqa_webui.pm]: Yes, and I'm proposing #85 separately.

**Review by perlpunk — APPROVED** [review]

**Comment by okurz** [comment]
This looks fine and I prefer it over #86 which can't work within os-autoinst-distri-openQA anyway.

**Comment by kalikiana** [comment]
> This looks fine and I prefer it over #86 which can't work within os-autoinst-distri-openQA anyway.

It does work. Please keep an open mind when I am trying something out which takes about as much time as the whole conversation 😃 

Anyway fine by me to go with this version.

**Comment by okurz** [comment]
Yes, you fixed it by including a valid definition of `script_retry` now

**Pull request merged**